kubectl-pg/cmd/extVolume.go

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright © 2019 NAME HERE <EMAIL ADDRESS>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cmd
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+	PostgresqlLister "github.com/zalando/postgres-operator/pkg/generated/clientset/versioned/typed/acid.zalan.do/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"log"
+)
+
+// extVolumeCmd represents the extVolume command
+var extVolumeCmd = &cobra.Command{
+	Use:   "ext-volume",
+	Short: "Increases the volume size of a given Postgres cluster",
+	Long:  `Extends the volume of the postgres cluster. But volume cannot be shrinked.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		clusterName, _ := cmd.Flags().GetString("clusterName")
+		if len(args) > 0 {
+			volume := args[0]
+			extVolume(volume, clusterName)
+		}
+	},
+	Example: "kubectl pg ext-volume [VOLUME] -c [CLUSTER-NAME]",
```

Comment:
does `[VOLUME]` here mean `[NEW-VOLUME-SIZE]` ?

Reply:
Yes, I will update as mentioned.

Reply:
please also add examples with different units of volume size and mention possible sizes (or a link to them) in the help message